tools/donate-cpu.py

```diff
@@ -35,454 +35,8 @@
 import re
 import tarfile
 import platform
+from donate_cpu_lib import *
 
-
```

Comment:
All functions moved to separate file to simplify re-usage. They almost identical, except get package function was extended w/ keeping backward compatibility and added one new function